src/Layers/xrRenderPC_R2/r2.h

```diff
@@ -68,6 +68,8 @@ class CRender : public D3DXRenderBase
         u32 disasm : 1;
         u32 advancedpp : 1; //	advanced post process (DOF, SSAO, volumetrics, etc.)
 
+        u32 no_ram_textures : 1; // don't keep textures in RAM
```

Comment:
Please, move it upper, after `nulltr`. It's reasonable to separate just technical features from graphical features.